### PR TITLE
#1853: Remove the warning of wrong browser type and version in Menas

### PR DESCRIPTION
**Pull request by benedeki**
Just a simple removal of a dialog popping up, if conditions are met on Menas opening.

_Release notes suggestion:_
Warning of non-Chrome browsers reworked and reworded. Multiple other browsers should work now without warning: Firefox 78+, Edge 79+, Safari 12+.

### CODE REVIEW

**Review by dk1844 — APPROVED** [review]
- [x] code reviewed
- [x] pulled
- [x] built
- [x] run

LGTM, works as expected.

**Review by AdrianOlosutean — DISMISSED** [review]

**Comment by benedeki** [comment]
Put the warning back, but little more elaborate. Would welcome a test on Safari.

File: menas/ui/components/login/loginDetail.controller.js [hunk 1]
```diff
@@ -73,8 +73,16 @@ sap.ui.define([
     },
 
     onAfterRendering: function() {
-      if(!sap.ui.Device.browser.chrome || (sap.ui.Device.browser.chrome && sap.ui.Device.browser.version < 68)) {
-        sap.m.MessageBox.warning("Chrome (version 68 and higher) is currently the only supported browser for menas");
+      if (sap.ui.Device.browser.chrome && sap.ui.Device.browser.version < 68) {
+        sap.m.MessageBox.warning("Chrome browser version lower then 68 is known of having issue displaying the Menas application");
```
Comment (AdrianOlosutean) [inline menas/ui/components/login/loginDetail.controller.js:77]:
```suggestion
        sap.m.MessageBox.warning("Chrome browser version lower than 68 is known of having issue displaying the Menas application");
```

File: menas/ui/components/login/loginDetail.controller.js [hunk 2]
```diff
@@ -73,8 +73,16 @@ sap.ui.define([
...
+      } else if (sap.ui.Device.browser.firefox && sap.ui.Device.browser.version < 78) {
+        sap.m.MessageBox.warning("Firefox browser version lower then 78 is known of having issue displaying the Menas application");
```
Comment (AdrianOlosutean) [inline menas/ui/components/login/loginDetail.controller.js:79]:
```suggestion
        sap.m.MessageBox.warning("Firefox browser version lower than 78 is known of having issue displaying the Menas application");
```

File: menas/ui/components/login/loginDetail.controller.js [hunk 3]
```diff
@@ -73,8 +73,16 @@ sap.ui.define([
...
+      } else if (sap.ui.Device.browser.edge && sap.ui.Device.browser.version < 79) {
+        sap.m.MessageBox.warning("Edge browser version lower then 79 is known of having issue displaying the Menas application");
```
Comment (AdrianOlosutean) [inline menas/ui/components/login/loginDetail.controller.js:81]:
```suggestion
        sap.m.MessageBox.warning("Edge browser version lower than 79 is known of having issue displaying the Menas application");
```

File: menas/ui/components/login/loginDetail.controller.js [hunk 4]
```diff
@@ -73,8 +73,16 @@ sap.ui.define([
...
+      } else if (sap.ui.Device.browser.safari && sap.ui.Device.browser.version < 12) {
+        sap.m.MessageBox.warning("Safari browser version lower then 12 is known of having issue displaying the Menas application");
```
Comment (AdrianOlosutean) [inline menas/ui/components/login/loginDetail.controller.js:83]:
```suggestion
        sap.m.MessageBox.warning("Safari browser version lower than 12 is known of having issue displaying the Menas application");
```

**Review by AdrianOlosutean — CHANGES_REQUESTED** [review]
I will test safari too

**Review by AdrianOlosutean — APPROVED** [review]
Tested with a higher version of Safari and no warning

**Review by Zejnilovic — APPROVED** [review]

**Pull request merged**